tests/nnfw_api/lib/GenModelTrain.h

```diff
@@ -132,6 +131,17 @@ struct TrainCaseData
     std::memcpy(dest.back().data(), data.data(), size);
   }
 
+  /**
+   * @brief Add float data to a vector
+   *
+   * @param dest Destination to store vector data
+   * @param data A vector data to be stored
```

Comment:
It would be better to use "loss" explicitly.
```suggestion
   * @brief Add loss data as a float vector
   *
   * @param dest Destination where loss data will be stored
   * @param data A loss data to be stored
```